File: src/main/java/com/onysakura/tools/url/UrlEntity.java
package com.onysakura.tools.url;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class UrlEntity {

    @Id
    @Column(length = 10, columnDefinition = "varchar(10) COMMENT '编码'")
    private String code;

    @Column(length = 1024, columnDefinition = "varchar(1024) COMMENT '原始地址'")
    private String url;

    public UrlEntity() {
    }

    public UrlEntity(String code, String url) {
        this.code = code;
        this.url = url;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: src/main/kotlin/com/onysakura/tools/url/UrlEntity.kt
package com.onysakura.tools.url

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id

@Entity
class UrlEntity() {
    @Id
    @Column(length = 10, columnDefinition = "varchar(10) COMMENT '编码'")
    var code: String = ""

    @Column(length = 1024, columnDefinition = "varchar(1024) COMMENT '原始地址'")
    var url: String = ""

    constructor(code: String, url: String) : this() {
        this.code = code
        this.url = url
    }

    override fun toString(): String {
        return "UrlEntity(code='$code', url='$url')"
    }
}